docs/reference/concepts/snapshots.md

```diff
@@ -2,32 +2,52 @@
+A simple example of generating a snapshot:
 
-In the project’s [`Editor/SnapshotGenerator`](https://github.com/spatialos/UnityGDK/tree/master/workers/unity/Assets/Playground/Editor/SnapshotGenerator) folder, there is a simple example of generating a snapshot through code.
+```csharp
+public static void GenerateSnapshot() 
+{
+    // Create a snapshot object
+    var snapshot = new Snapshot();
 
-You can use this as a base for your own project’s snapshot generation by copying the file to the same folder in your own project and editing the `SnapshotGenerator` class to add SpatialOS entities to a snapshot.
+    // Create a template...
+    var template = new EntityTemplate();
+    template.AddComponent(new Position.Snapshot(playerSpawnerLocation), WorkerUtils.UnityGameLogic);
+    template.AddComponent(new Metadata.Snapshot { EntityType = "Cube" }, WorkerUtils.UnityGameLogic);
+    template.AddComponent(new Persistence.Snapshot(), WorkerUtils.UnityGameLogic);
+    template.SetReadAccess(WorkerUtils.UnityGameLogic, WorkerUtils.UnityClient, WorkerUtils.MobileClient);
+    template.SetComponentWriteAccess(EntityAcl.ComponentId, WorkerUtils.UnityGameLogic);
 
-### Generate the snapshot
+    // ..and add it to the snapshot.
+    snapshot.AddEntity(template);
 
-To generate the snapshot, in your Unity Editor menu: **SpatialOS** > **Generate snapshot** to open the snapshot generator window, then select `Generate snapshot` there.
+    // WriteToFile operates realtive to your working directory.
+    // In the Unity Editor, this is the Assets directory of your Unity project.
+    snapshot.WriteToFile("../../../snapshots/default.snapshot");
+}
+```
 
-This saves the generated snapshot to `snapshots/default.snapshot`, which is where SpatialOS expects to find it unless explicitly told to use another path when you start the deployment.
+A common usage pattern is to expose the snapshot generation through a Unity Editor menu item or window. You can see an example of this in the [FPS Starter Project](https://github.com/spatialos/gdk-for-unity-fps-starter-project/blob/<%(Var key="current_version")%>/workers/unity/Assets/Fps/Scripts/Editor/SnapshotGenerator/SnapshotMenu.cs). This allows you to quickly generate a snapshot with a few clicks!
 
 ## How to start a deployment from a snapshot
 
-You can start local or cloud deployments using the `spatial local launch` and `spatial cloud launch` commands respectively - see the documentation on [Deploying your game]({{urlRoot}}/reference/concepts/deployments) for details.
+You can start a local deployment via the Unity Editor menu: **SpatialOS** > **Local launch** (Ctrl+L/Cmd+L). This starts a deployment with the snapshot found at `<your_spatialos_project>/snapshots/default.snapshot`. If you wish to launch via the CLI or with a different snapshot, see the section below.
```

Comment:
is it meant to be `your_spatialos_project` or `spatialos_project_root`?

Reply:
The latter is probably clearer